src/index.js

```diff
@@ -7,6 +7,8 @@ export { default as PlotlyLineChart } from './components/plotly/line'
 export { default as PlotlyScatterChart } from './components/plotly/scatter'
 export { default as PlotlyPieChart } from './components/plotly/pie'
 export { default as PlotlyPyramidChart } from './components/plotly/pyramid-bar'
+export { default as PlotlyBarLineChart } from './components/plotly/bar-line'
```

Comment:
@kc-leung please make 2 commits out of this, the 2 file changes should not have been put together.